src/Unicorn/ControlPanel/Controls/ConfigurationDetails.cs

```diff
@@ -1,115 +1,121 @@
-
-			if (collapse) writer.Write(@"
-				</div>");
-		}
+    /// <summary>
+    ///     Renders the current dependency/provider configuration for Unicorn, using IDocumentable to show additional details
+    ///     when available.
+    /// </summary>
+    internal class ConfigurationDetails : IControlPanelControl
+    {
+        private readonly IConfiguration _configuration;
+        private readonly IPredicate _predicate;
+        private readonly ITargetDataStore _serializationStore;
+        private readonly ISourceDataStore _sourceDataStore;
+        private readonly IEvaluator _evaluator;
+
+        public ConfigurationDetails(IConfiguration configuration, IPredicate predicate, ITargetDataStore serializationStore, ISourceDataStore sourceDataStore, IEvaluator evaluator)
+        {
+            _configuration = configuration;
+            _predicate = predicate;
+            _serializationStore = serializationStore;
+            _sourceDataStore = sourceDataStore;
+            _evaluator = evaluator;
+        }
+
+        public string ConfigurationName { get; set; }
+        public string ModalId { get; set; }
+
+        public void Render(HtmlTextWriter writer)
+        {
+            var collapse = !string.IsNullOrWhiteSpace(ModalId);
+
+            if (collapse)
+                writer.Write(@"<div id=""{0}"" class=""overlay"">", ModalId);
+
+            writer.Write(@"<article class=""modal"">");
 
-		private void RenderType(bool collapsed, string categorization, string categoryDescription, object type, HtmlTextWriter writer)
-		{
-			writer.Write(@"
+            if (collapse)
+                writer.Write(@"<h2>{0} Details</h2>", ConfigurationName);
+
+            RenderDependencies(writer);
+
+            RenderType(collapse, "Predicate", "Predicates define which items are included or excluded in Unicorn.", _predicate, writer);
+
+            RenderType(collapse, "Target Data Store", "Defines how items are serialized, for example to disk using YAML format.", _serializationStore, writer);
+
+            RenderType(collapse, "Source Data Store", "Defines how source data is read to compare with serialized data. Normally this is a Sitecore data store.", _sourceDataStore, writer);
+
+            RenderType(collapse, "Evaluator", "The evaluator decides what to do when included items need updating, creation, or deletion.", _evaluator, writer);
+
+            writer.Write(@"</article>");
+
+            if (collapse)
+                writer.Write(@"
+				</div>");
+        }
+
+        private void RenderDependencies(HtmlTextWriter writer)
+        {
+            var dependencies = ControlPanelUtility.FindConfigurationsDependencies(_configuration).ToArray();
+            if (dependencies.Any())
+            {
+                writer.Write(@"<h3>Dependencies</h3>");
+                writer.Write($@"<p class=""help"">This configuration is dependent on items in other configurations. Please review the following before synchronizing:</p>");
```

Comment:
Use of 'dependent' (en-US) and 'dependant' (en-UK) is inconsistent in text and variables. I'm fine with either one.